sphinx_rtd_theme/footer.html

```diff
@@ -32,5 +32,9 @@
   {% trans %}Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>{% endtrans %}.
   {%- endif %}
 
+  {%- if commit %}
+  Build commit: <code>{{ commit }}</code>
```

Comment:
I wouldn't want debug info in the flyout, but having a link to the build is what i was imagining in the flyout. That is, replace "Builds" with "Build", as "Builds" really isn't useful. Okay to put this in the footer though.

For VCS support, displaying the commit will only happen on RTD or maybe eventually those building docs with our extension, if we move that logic there. So, I'm happy if the truncating logic happens in RTD.

Because this patch is only useable with RTD at the moment, I'm fine linking to the build as well. If not on RTD, just degrade to no link, but display the revision if we have it.

Anyways, as for the display now, it looks hacky:
![screen shot 2015-07-17 at 4 59 01 pm](https://cloud.githubusercontent.com/assets/1140183/8759282/a17a3dd4-2ca5-11e5-8377-4ffe92a5cf33.png)

I'd prefer to put this in the line above, where the last modified date would normally go, and make some changes to wording. Here's what I have when we have a build:

![screen shot 2015-07-17 at 6 20 30 pm](https://cloud.githubusercontent.com/assets/1140183/8759738/cd1cb772-2cb0-11e5-8376-334afc86dfa3.png)

Here's no build, and only a revision:
![screen shot 2015-07-17 at 6 37 01 pm](https://cloud.githubusercontent.com/assets/1140183/8759797/dc6e123c-2cb2-11e5-9ad9-07536fce477b.png)